index.html

```diff
@@ -56,6 +56,148 @@ <h2>I'm Cristhian. Glad to see you!</h2>
           </ul>
         </div>
       </section>
+      <section class="section-p2">
+        <div class="cards-container">
+          <div class="card">
+            <img class="img-project" src="/img/Snapshoot-Portfolio1.svg" alt="">
+            <div class="description-project tonic">
+              <h3>Tonic</h3>
+              <div class="frame">
+                <p>CANOPY</p>
+                <img src="/img/Counter.svg">
+                <p>Back End Dev</p>
+                <img src="/img/Counter.svg">
+                <p>2015</p>
+              </div>
+              <p class="text-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="technologys">
+                <ul class="list-technologys">
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+                </ul>
+              </div>
+              <button type="button">See project</button>
```

Comment:
- [ ] kindly consider giving this button the button styling: Active, Hover, and Pressed as given in the Figma design to give a good user experience